src/test/scala/edu/colorado/plv/cuanto/numdomain/NumDomainSpec.scala

```diff
@@ -0,0 +1,140 @@
+package edu.colorado.plv.cuanto.numdomain
+
+import apron.{Box, Manager, Octagon, Polka, _}
+import gmp.Mpfr
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.util.{Try, Failure, Success}
+
+/**
+  * Created by Tianhan Lu on 4/20/17.
```

Comment:
Nit, but please follow what you agreed to: https://cuplv.slack.com/archives/C3S635ZD4/p1497045800481119